coresimd/powerpc/altivec.rs

```diff
@@ -655,6 +701,28 @@ where
     a.vec_add(b)
 }
 
+/// Vector permute.
+#[inline]
+#[target_feature(enable = "altivec")]
+pub unsafe fn vec_perm<T>(a: T, b: T, c: vector_unsigned_char) -> T
+where
+    T: sealed::VectorPerm,
+{
+
+    if cfg!(target_endian = "little") {
```

Comment:
prefer conditional compilation here: `#[cfg(target_endian = "little")]` and `#[cfg(not(target_endian = "little"))]` (or `#[cfg(target_endian = "big")]`).

Reply:
Something along the lines of
``` rust
#[cfg(target_endian = "little")]
mod endian {

}
#[cfg(target_endian = "big")]
mod endian {

}
use self::endian::*;
```